charts/orchestrator/templates/release-candidate.yaml

```diff
@@ -0,0 +1,104 @@
+{{/*
+This file used for including source resources for RHDH and Sonataflow components from RC builds.
+It requires including a secret to enable access to the image registries that contains the builds.
```

Comment:
Should we check for the secret to exist before installing the chart? So to avoid the situation where the secret doesn't exist but we install the chart and the deployment fails... WDYT?

Reply:
I don't think we should check for it. There is a document for QE how to enable their clusters to work with the registry for CR builds from brew.
The credentials should be added to the secret openshift-config/pull-secret which rolls its changes to all of the nodes.

Reply:
ack